src/site/stages/build/drupal/graphql/homePage.graphql.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * Home page
+ */
+
+const menu = 'homepage-top-tasks-blocks';
+const hubListQueue = 'home_page_hub_list';
+const promoBlocksQueue = 'home_page_promos';
+
+const {
+  featureFlags,
+  enabledFeatureFlags,
+} = require('../../../../utilities/featureFlags');
+
+const query = `
```

Comment:
I'm not sure how to verify all this is correct, so...I hope it is! :see_no_evil:

What would happen if something's off in here? (That is to ask, how can I be sure this is golden?)